src/pug/sidebar-new.pug

```diff
@@ -1,5 +1,12 @@
 - var i = 0;
 aside.pf-c-page__sidebar.pf-l-page__sidebar#ins-c-sidebar
+    .pf-l-split.ins-c-navigation__header
+        .pf-l-split__item.ins-c-page__home-icon
+            a
+                svg(style='vertical-align: -0.1875em;' fill='currentColor' height='1.5em' width='1.5em' viewbox='0 0 576 512' aria-hidden='true' role='img')
```

Comment:
Since the Home icon is also used once the page is loaded, can we put the svg in /static/images and reference it in both locations?

Reply:
We _could_, but the home icon is actually coming from the Patternfly/react-icons library

Reply:
@kruai Updated to just import the icon from font-awesome (which PF provides)